components/studio/bin/hab-studio.sh

```diff
@@ -909,6 +909,19 @@ exit_with() {
   exit $2
 }
 
+# **Internal** Builds up a secret environmt based on the prefix `HAB_STUDIO_SECRET_`
```

Comment:
environmt

Reply:
you have to say it with a Texas twang to make sense ;)

Reply:
en-*var*-mint